src/Components/Main.tsx

```diff
@@ -1,13 +1,14 @@
 import { useEffect, useState } from "react";
-import { useDispatch} from 'react-redux'
+import { useDispatch} from 'react-redux';
 import { getWeather, getApiAdress, getGeolocation } from '../Api';
-import SearchBar from "./SearchBar";
 import { WeatherDataType as TypesWeatherData } from "../Types/weather";
+import SearchBar from "./SearchBar";
 import LeftSideBar from "./LeftSideBar";
 import Content from "./Content";
 
 const Main = () => {
     const [weatherData, setWeatherData] = useState<TypesWeatherData>();
```

Comment:
You have an unused state variable `weatherData`. BTW always take a look at the terminal, it shows warnings.